datasets/search.md

```diff
@@ -0,0 +1,126 @@
+is heavily restricted in re:dash.
+We also maintain a restricted group for search on Github and Bugzilla.
+If you reach a 404 on Github or don't have access to a re:dash query or bug
+this is likely your issue.
+To get access permissions, file a bug using the [search permissions template]
+
+Once you have proper permissions,
+you'll have access to a new source in re:dash called `Presto Search`.
+**You will not be able to access any of the search datasets
+via the standard `Presto` datasource**, even with proper permissions.
+
+
+# Terminology
+
+## Direct vs Follow-on Search
+
+Searches can be split into two major classes `direct` and `follow-on`.
+
+Direct searches result from a direct interaction with a `search access point` (SAP).
+These searches are often called SAP searches.
+There are currently 6 SAPs:
+
+* `urlbar`
+* `searchbar`
+* `newtab`
+* `abouthome`
+* `contextmenu`
+* `system`
+
+Users will often interact with the Search Engine Results Page (SERP)
+to create "downstream" queries.
+These queries are called `follow-on queries`.
+For example, follow-on queries can be caused by:
+
+* Revising a query (`restaurants` becomes `restaurants near me`)
+* Clicking on the "next" button
+* Accepting spelling suggestions
+
+## Tagged vs Untagged Searches
+
+Our partners (search engines) attribute queries to Mozilla using **partner codes**.
+When a user issues a query through one of our SAPs,
+we include our partner code in the URL of the resulting search.
+
+**Tagged queries** are queries that **include one of our partner codes**.
+
+**Untagged queries** are queries that **do not include one of our partner codes**.
+If a query is untagged,
+it's usually because we do not have a partner deal for that search engine and region.
+
+If an SAP query is tagged, any follow-on query should also be tagged.
+
+# Standard Search Aggregates
+
+We report three types of searches in our search datasets:
+`SAP`, `tagged-sap`, and `tagged-follow-on`.
+These aggregates show up as columns in the
+`search_aggregates` and `search_clients_daily` datasets.
+
+The **`SAP` column counts all SAP (or direct) searches**.
+`SAP` search counts are collected via `UI telemetry`.
+`UI telemetry` refers to the probes integrated with the Firefox UI.
+`UI telemetry`, and thus `SAP` search counts,
+ are **very reliable, but do not count follow-on queries**.
+
+In 2017-06 we deployed the [followonsearch addon],
+which adds probes for `tagged-sap` and `tagged-follow-on` searches.
+These columns **attempt to count all tagged searches**
+by looking for Mozilla partner codes in the URL of requests to partner search engines.
+These search counts are critical to understanding revenue
+since they exclude untagged searches and include follow-on searches.
+However, these search counts have **important caveats affecting their reliability**.
+See [In Content Telemetry Issues](#in-content-telemetry-issues) for more information.
+
+# In Content Telemetry Issues
```

Comment:
Do we really need a whole section on this? It seems appropriate to just include the content here as an extension of the discussion of the follow-on search counts, immediately above.

Reply:
Yeah, I'm torn on this. I decided to include a header because:
* I want the TOC to show this section prominently
* I want to be able to link to this section specifically. This has caused confusion in the past and I want to be able to redirect questions.

But I'm happy to remove if you think it's too disruptive.